locales/en-US/snoozetabs.properties

```diff
@@ -20,9 +20,9 @@ timeUpcomingNextOpen=the next time Firefox opens
 timeUpcomingToday=later today at {time}
 # "{time}" is the time the tab will snooze until.
 timeUpcomingTomorrow=tomorrow at {time}
-# "{weekday}" is the day of the week, "{day}" is the day of the month.
+# "{date}" is the full date of the month.
 # Example: "Sun, Mar 5 at 7:05am".
-timeUpcomingLater={weekday}, {month} {day} at {time}
+timeUpcomingOther={date} at {time}
```

Comment:
This is great to see as this pattern corresponds 1-1 to a pattern in CLDR that we currently don't expose, but hopefully will one day :)
[0] http://www.unicode.org/cldr/charts/30/summary/en.html#1945